login/service.go

```diff
@@ -608,56 +681,69 @@ func encodeToken(ctx context.Context, referrer *url.URL, outhToken *oauth2.Token
 }
 
 // CreateOrUpdateKeycloakUser creates a user and a keycloak identity. If the user and identity already exist then update them.
-func (keycloak *KeycloakOAuthProvider) CreateOrUpdateKeycloakUser(accessToken string, ctx context.Context, profileEndpoint string) (*account.Identity, *account.User, error) {
-	var identity *account.Identity
-	var user *account.User
+func (keycloak *KeycloakOAuthProvider) CreateOrUpdateKeycloakUser(accessToken string, ctx context.Context, profileEndpoint string, user *account.User, identity *account.Identity) error {
```

Comment:
Maybe it's better to call these parameters as "witUser" and "witIdentity" to avoid confusion (which can be nil) and keep returning *Identity, *User
These parameters are temporal. We won't need to pass wit users/identities after successful migration. So we will remove these parameters. Later this week I hope :)

Reply:
done in d596a03

Reply:
Parameters still have the old confusing names instead of proposed witIdentity, witUser.